apps/charterafrica/src/components/GrantsAndFellowshipsGrid/GrantsAndFellowshipsGrid.js

```diff
@@ -0,0 +1,96 @@
+import { Section } from "@commons-ui/core";
+import { Typography, Box, Divider } from "@mui/material";
+import React from "react";
+
+import OpportunityCardList from "../OpportunityCardList";
+
+import { neutral, secondary } from "@/charterafrica/colors";
+
+const GrantsAndFellowshipsGrid = React.forwardRef(
+  function GrantsAndFellowshipsGrid(props, ref) {
+    const { grants, title, sx } = props;
+
+    if (!grants) {
+      return null;
+    }
+
+    // group grants by status
+    const grantsByStatus = grants.reduce((acc, grant) => {
+      const { status } = grant;
+      if (acc[status]) {
+        acc[status].push(grant);
+      } else {
+        acc[status] = [grant];
+      }
+      return acc;
+    }, {});
+
+    const grantsByStatusArray = Object.keys(grantsByStatus).map((key) => {
+      return {
+        title: `${key} calls`,
+        grants: grantsByStatus[key],
+      };
+    });
+
+    return (
+      <Box
+        sx={{
+          backgroundColor: secondary[50],
+          ...sx,
+        }}
+        ref={ref}
+      >
+        <Section
+          sx={{
+            px: { xs: 5, sm: 0 },
+            py: 5,
+            // hide from 2nd onwards on mobile and tablet
+            "& > *:nth-of-type(n+2)": {
+              display: {
+                xs: "none",
+                md: "block",
+              },
+            },
```

Comment:
For slow ones such as my self, why are we hiding them?

Reply:
On `mobile` and `tablet` screens, you show only one of the groups, i.e open call on grants, and technologies only on fellowships. This hides the rest of the groups and shows them on desktop